command/src/state.rs

```diff
@@ -332,6 +332,17 @@ impl ConfigState {
   }
 }
 
+pub fn get_application_id_by_domain(state: &ConfigState, domain: &str) -> Option<String> {
+  state.http_fronts.values()
+    .find(|fronts| fronts.iter().find(|front| &format!("{}{}", front.hostname, front.path_begin) == domain).is_some())
+    .map(|front| front.first().unwrap().app_id.clone())
```

Comment:
I think you could get multiple apps for one domain, so taking the first is not enough

Reply:
I wasn't sure about this one, yes. I'll see to change that